app/controllers/users_controller.rb

```diff
@@ -1,7 +1,66 @@
 class UsersController < ApplicationController
 
+  require 'beard_trimmer'
+  require 'beard_trimmer/with_massage'
   def list
     @users = User.all
   end
+  
+  # This function return all users who have massaged
+  def find_massage
```

Comment:
Would have liked to see this moved into the `update` action, with 'massage', 'trim' and 'shave' just be parameters in the `put` request.